.github/workflows/publish.yaml

```diff
@@ -36,20 +36,21 @@ jobs:
             files.pythonhosted.org:443
             ghcr.io:443
             pkg-containers.githubusercontent.com:443
+            cdn02.quay.io:443
```

Comment:
I'd suggest moving to the other quay.io endpoint above. I think genereally, that sorting these based on the domains would be helpful.

Reply:
Good idea, will sort them. I didn't have a choice of domains. The manylinux container downloads from quay.io registry and I suspect maturin uses the cdn. I'll make sure to add all the cdns for quay.io